src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
```

Comment:
you need to override `toString` for this class, since you use it in exception messages